Google.PowerShell/Compute/GceHealthCheckCmdlets.cs

```diff
@@ -16,24 +16,20 @@ namespace Google.PowerShell.ComputeEngine
     /// https://cloud.google.com/compute/docs/load-balancing/health-checks
     /// </para>
     /// <example>
-    ///   <para> Adds an HTTP health to project "my-project". </para>
-    ///   <para><code> PS C:&gt; Add-GceHealthCheck "my-health-check" -Project "my-project" -Http </code></para>
+    ///     <code> PS C:&gt; Add-GceHealthCheck "my-health-check" -Project "my-project" -Http </code>
+    ///     <para> Adds an HTTP health check to the project named "my-project". </para>
     /// </example>
     /// <example>
-    ///   <para> Adds an HTTP health to project "my-project". </para>
-    ///   <para><code> PS C:$gt; Add-GceHealthCheck "my-health-check" -Project "my-project" -Http </code></para>
+    ///     <code> PS C:&gt; Add-GceHealthCheck "my-health-check" -Https </code>
+    ///     <para> Adds an HTTPS health check to the project in the Cloud SDK config. </para>
     /// </example>
     /// <example>
-    ///   <para> Adds an HTTPS health to the project in the Cloud SDK config. </para>
-    ///   <para><code> PS C:&gt; Add-GceHealthCheck "my-health-check" -Https </code></para>
-    /// </example>
-    /// <example>
-    ///   <para> Adds an HTTP health with non-default values</para>
-    ///   <para><code>
-    ///     PS C:&gt; Add-GceHealthCheck "my-health-check" -Http -Description "Description of my health check." `<br/>
-    ///         -HostHeader "mydomain.com" -Port 50 -RequestPath "/some/path" -CheckInterval "0:0:2" `<br/>
-    ///         -Timeout "0:0:2" -HealthyThreshold 3 -UnhealthyThreshold 3 `<br/>
-    ///   </code></para>
+    ///   <code>
+    ///     <para>PS C:&gt; Add-GceHealthCheck "my-health-check" -Http -Description "Description of my health check." `</para>
+    ///     <para>            -HostHeader "mydomain.com" -Port 50 -RequestPath "/some/path" -CheckInterval "0:0:2" `</para>
+    ///     <para>            -Timeout "0:0:2" -HealthyThreshold 3 -UnhealthyThreshold 3 `</para>
+    ///   </code>
+    ///     <para> Adds an HTTP health check with non-default values</para>
```

Comment:
The convention other examples have been using is to have a description before the code snippet. Also, we have been using `<para><code>` not `<code><para>`. I don't know if that is the same, better, or worse. Just different.